### PR TITLE
Connect Intent To Service

### PR DESCRIPTION
**Pull request by 1AhmedYasser**
Bugs Related:

- https://github.com/buerokratt/Service-Module/issues/30#issuecomment-1978789313
- https://github.com/buerokratt/Service-Module/issues/30#issuecomment-1978794876
- https://github.com/buerokratt/Service-Module/issues/196#issuecomment-1978692236

Based on Tasks:

- https://github.com/buerokratt/Service-Module/issues/196
- https://github.com/buerokratt/Service-Module/issues/30

### CODE REVIEW

**Comment by baha-a** [comment]
Also Related to: 
- #362

File: DSL/Resql/services/get-services.sql
Comment (ExiRain) [inline DSL/Resql/services/get-services.sql:1]: Wouldn't if fail without ; on the end of SQL?

Comment (baha-a) [reply DSL/Resql/services/get-services.sql:1]: PostgreSQL will not fail if the SQL is missing a trailing semicolon.

**Review by ExiRain — APPROVED** [review]
Looks ok.

**Pull request merged**